calculator.js

```diff
@@ -2,8 +2,55 @@
 
 export class Calculator {
   //write your code here
+  constructor(num1){
+    this.num1 = num1;
+
+  }
+
+  perkalian(num2){
+    return this.num1 * num2;
```

Comment:
butuh di revisi sedikit lagi, agar method chaining nya jalan.
yg di return adalah object nya yaitu this, sehingga bisa dipanggil dengan cara : 
console.log(kali.perkalian(10).pengurangan(2));